src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,35 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import helpers from '../helpers';
+import services from '../services';
+
+dotenv.config();
+
+const { SECRET_KEY } = process.env;
+
+const { responseMessage } = helpers;
+const { userServices: { findUser } } = services;
+
+export default (request, response, next) => {
+  const token = request.headers.authorization || request.query.token;
```

Comment:
Great work!

I will like to suggest that you search for Bearer prepend in your token and take it off i.e "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwiaWF0IjoxNTY0ODI3MzI1LCJleHAiOjE1NjU0MzIxMjV9.p1yXbgzLUPGtF73o5p-ASLqnYVnTX4efUECtGlawtN4" This is standard and it helps to differentiate different types of a token in a larger application. What do you think?

Reply:
Yeah I understand your point, and I appreciate your feedback. However, I considered the context of our product, and like we discussed this morning, I feel it won't be necessary to prepend the token with 'Bearer' and still go ahead to remove it in the code. I feel it's more straightforward this way.

Reply:
ok,

Reply:
LGTM